tvmaze/show.go

```diff
@@ -106,6 +107,15 @@ func (s Show) GetIMDBID() string {
 	return val
 }
 
+func (c Client) GetShows(offset int) ([]Show, error) {
+	url := baseURLWithPathQuery("shows", "page", strconv.Itoa(offset))
+	shows := []Show{}
+	if err := c.get(url, &shows); err != nil {
```

Comment:
The API docs mention that when you go past the last page, the endpoint returns a 404. Can we handle that gracefully to avoid propagating the error back up here? I'm thinking it could return `nil, nil` in the case that we've reached the end of the results.

Reply:
Sure. This will be a little more involved though -- currently the status code isn't considered by the client. I can return the StatusCode via `get` and check for 404s -- they are also returned by a few other shows apis (and ignored currently).

With the current code, this comes through as `[]Show{}, nil`